tests/plugins/commit-message/index.js

```diff
@@ -174,6 +178,18 @@ describe("commit-message", () => {
                 await emitBotEvent(bot, { action, pull_request: { number: 1, title: `Update: ${"A".repeat(72)}` } });
                 expect(nockScope.isDone()).toBeTruthy();
             });
+
+            test("Does not post a status if the repository is excluded", async() => {
```

Comment:
No assert for this unit test?

Reply:
Since `nock.disableNetConnect();` is being used, the test would fail if any HTTP requests were sent (or if `emitBotEvent` threw an error.)